test/test_cost_functions.jl

```diff
@@ -11,3 +11,56 @@ res = [1.0, 0.1, 0.1]
 for (ix, v) in enumerate(bps)
     @test isapprox(v, res[ix])
 end
+
+@testset "Test MarketBidCost with Polynomial Cost Timeseries with Service Forecast " begin
```

Comment:
We need a test that serializes and de-serializes a system with MarketBidCost. It should just work but it is better to test it.

Reply:
I'll add that test